tests/unit/test_interface.py

```diff
@@ -215,6 +474,34 @@ def test_canonical_interface_name_failure():
         interface.canonical_interface_name(**data)
 
 
```

Comment:
Might just add two test to make sure the ValueErrors (from my previous comment) are checked.

Reply:
I added a test to check that the `abbreviated_interface_name_list` function gives me the expected error. I had one for `canonical_interface_name_list` on line 488. Thanks Jeff!